_sass/page.scss

```
@@ -309,3 +309,45 @@ section.apply {
     margin-bottom: 1em;
   }
 }
+
+.dropbtn {
+  background-color: $light-blue;
+  color: white;
+  padding: 16px;
+  font-size: 20px;
+  border: none;
+  min-width: 140px;
+}
+  
+.dropdown {
+  position: relative;
+  display: inline-block;
+  margin-bottom: 16px;
+}
+  
+.dropdown-content {
+  display: none;
+  position: absolute;
+  background-color: #f1f1f1;
+  min-width: 180px;
+  z-index: 1;
+}
+  
+.dropdown-content a {
+  color: black;
+  padding: 12px 16px;
+  text-decoration: none;
+  display: block;
+}
```

Sass is available, so sass syntax should be used

```suggestion
.dropdown-content {
  display: none;
  position: absolute;
  background-color: #f1f1f1;
  min-width: 180px;
  z-index: 1;

  a {
    color: black;
    padding: 12px 16px;
    text-decoration: none;
    display: block;
  }
}
```

(and so forth with the other styles)

While not *strictly* necessary, all of the dropdown styles could also go under `.dropdown` in accordance with the hierarchy of the related elements